hermes/bin/src/runtime_extensions/wasi/filesystem/host.rs

```diff
@@ -424,6 +424,6 @@ impl filesystem::preopens::Host for HermesRuntimeContext {
     fn get_directories(
         &mut self,
     ) -> wasmtime::Result<Vec<(wasmtime::component::Resource<Descriptor>, String)>> {
-        todo!()
```

Comment:
These Filesystem functions ultimately need implementations, put TODO comments so we can find them easier.